fenix/app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -207,6 +211,15 @@ class HistoryFragment : LibraryPageFragment<History>(), UserInteractionHandler,
         }
     }
 
+    private fun updateDeleteMenuItemView(isEmpty: Boolean) {
```

Comment:
nit: can we update 'isEmpty' to something like 'enabled' or ''isEnabled' then update callers as required